ccmlib/scylla_node.py

```diff
@@ -1330,45 +1330,69 @@ def upgrade(self, upgrade_to_version):
     def rollback(self, upgrade_to_version):
         self.upgrader.upgrade(upgrade_version=upgrade_to_version, recover_system_tables=True)
 
+    def hostid(self, timeout=60, force_refresh=False):
+        if hasattr(self, 'node_hostid') and not force_refresh:
+            return self.node_hostid
+        node_address = self.address()
+        url = f"http://{node_address}:10000/storage_service/hostid"
+        response = requests.get(url=url)
+        if response.status_code == requests.codes.ok:
+            self.debug(f"/storage_service/hostid: {response.json()}")
+            for r in response.json():
```

Comment:
This API can return multiple host_ids ?

On which cases ?

Reply:
It's name is misleading, it usually returns multiple endpoints/hostsids, of all nodes which are normal token owners, unless it's a single node cluster.

Reply:
a bit sad there's no API for getting this node host_id...

Reply:
Oops, I looked at the jmx code and there is one I forgot about (that being used by `nodetool info` indirectly via jmx.

Reply:
fixed in v3